Packs/CommonPlaybooks/ReleaseNotes/2_3_54.md

```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Indicator Registration Polling - Generic
+- Added the playbook to the xpanse marketplace.
```

Comment:
```suggestion
<!-- 
##### Indicator Registration Polling - Generic
- Added the playbook to the xpanse marketplace.
-->
```
Ideally we'd want this to appear only in XPANSE RNs but the <~XPANSE> tag isn't supported yet. So better just to hide this RN